Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -173,11 +173,11 @@ public bool UpdateSolvers
         /// <summary>
         /// List of solvers that this handler will manage and update
         /// </summary>
-        public IReadOnlyCollection<Solver> Solvers
+        public ICollection<Solver> Solvers
```

Comment:
Revert this and instead use the RegisterSolver and SetSolvers

Reply:
`IReadOnlyCollection` unfortunately does not have the `Contains` method, that was the reason for the change. Further the property still exposes a `set` method, that allows changing the list, so why offering the list as ReadOnly in the first place?